azure-pipelines.yml

```diff
@@ -156,6 +156,17 @@ jobs:
       HADOOP_HOME: $(Build.BinariesDirectory)\hadoop
       DOTNET_WORKER_DIR: $(Build.ArtifactStagingDirectory)\Microsoft.Spark.Worker\netcoreapp2.1\win-x64
 
+  - task: DotNetCoreCLI@2
+    displayName: 'E2E tests for Spark 3.0.0-preview'
+    inputs:
+      command: test
+      projects: '**/Microsoft.Spark*.E2ETest/*.csproj'
+      arguments: '--configuration $(buildConfiguration)'
+    env:
+      SPARK_HOME: $(Build.BinariesDirectory)\spark-3.0.0-preview-bin-hadoop2.7
```

Comment:
Looks like this has been [changed](https://www.apache.org/dyn/closer.lua/spark/spark-3.0.0-preview2/spark-3.0.0-preview2-bin-hadoop2.7.tgz) to `spark-3.0.0-preview2-bin-hadoop2.7.tar`.